teachopencadd/talktorials/019_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1250 @@
+{

```

Comment:
<p>I would name this file <code>minimized_topology.pdb</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>I see your point, but we would have to carry the name over to T020, since we are using the files there.  The minimized remark could cause some confusion as usually the files are simply called topology and trajectory and we do not explain minimization there. I am  undecided if it is worth, but happy to change it anyway. </p>

Reply:
Good point. You can leave it like that then!